SPECS/hwloc/hwloc.spec

```diff
@@ -1,66 +1,66 @@
-Summary:   Portable Hardware Locality - portable abstraction of hierarchical architectures
-Name:      hwloc
-Version:   2.0.4
-Release:   5%{?dist}
-License:   BSD
+Summary:        Portable Hardware Locality - portable abstraction of hierarchical architectures
+Name:           hwloc
+Version:        2.5.0
```

Comment:
There seems to be one test failure for this package. We currently want to have all packages added to core pass their tests, so please take a look at the failure.

Reply:
@PawelWMS, this same test is also failing in the fedora. I just build their latest rawhide hwloc. I think this is coming from the upstream. 
I did search for the upstream patch but could not find anything. 
Might just have to let it sit like this for now.  
![image](https://user-images.githubusercontent.com/106988478/217681212-ddaa52b1-d2d7-4a6b-b3ed-a74fe3dca5f2.png)
This is the same hwloc %check test that I ran on the fedora vm.

Reply:
@rikenm1, I think it actually succeeds in Fedora. At least it succeeded in their [latest build log for this package](https://kojipkgs.fedoraproject.org//packages/hwloc/2.5.0/5.fc38/data/logs/i686/build.log) (Fedora 38). I'd try digging deeper what might be causing the test to fail (check the logs or maybe there's a way to run the tests with more verbose output). Once we understand the failure, we can either correct it, or skip the test, if it's going to turn out, that it's not an issue on Mariner's end.

Reply:
this failure is only happening on the docker side. docker by default blocks some of the sys_calls. the syscall that is failing here is move_page()